docs/src/intro/publications.md

```diff
@@ -115,7 +138,7 @@ indicators with combination of process-based models and data assimilation techni
 Ecological Indicators 66, 428–439. <https://doi:10.1016/j.ecolind.2016.02.016>.
 
 
-## Theses
+### Theses & Internship reports
```

Comment:
Can we also add Laurène thesis there :) ?

Reply:
Good suggestion - it is now added to the list!